Badge.md

```diff
@@ -1,3 +1,139 @@
 (Version >= 1.16.0)
+With https://github.com/louislam/uptime-kuma/pull/1119
 
-Please read: https://github.com/louislam/uptime-kuma/pull/1119
\ No newline at end of file
+- [Description](#description)
+- [Open Badge Generator](#open-badge-generator)
+- [Badge Type Options](#badge-type-options)
+  - [Status badge](#status-badge)
+  - [Ping, uptime, avg-response and response badge](#ping-uptime-avg-response-and-response-badge)
+    - [Badge Duration at ping, uptime and avg-response](#badge-duration-at-ping-uptime-and-avg-response)
+  - [Certificate Expire badge](#certificate-expire-badge)
+- [Badge Styles](#badge-styles)
```

Comment:
Rationale: other pages of the wiki also don't have such a section

```suggestion
```

Reply:
you are right, a toc is no longer needed at github.com, there is the "Outline" feature, which creates the same thing.

[https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/)